packages/flutter_tools/test/asset_bundle_package_test.dart

```diff
@@ -133,6 +133,32 @@ $assetsSection
     }, overrides: <Type, Generator>{
       FileSystem: () => new MemoryFileSystem(),
     });
+    
+    testUsingContext('One package with one asset not specified', () async {
```

Comment:
The test names express conditions ("we are in situation Z"), but not the expected behavior ("what should the production code do in situation Z?"). That makes it harder to read the tests, because the reader has no clue about what behavior the test is trying to verify, and that information then needs to be gleaned from code. Maybe change to something like
```
group('Feature X', () {
  test('should do Y1 in situation Z1', () { ... });
  test('should do Y2 in situation Z2', () { ... });
});
```
If you are repeating yourself in the Ys or the Zs, consider nested groups, e.g.
```
group('Feature X', () {
  group('should do Y', () {
    test('in situation Z1', () { ... });
    test('in situation Z2', () { ... });
 });
});
```
Test execution output concatenates group and test names, so you'd get
```
...
01:52 +45: test/x_test.dart: Feature X should do Y in situation Z1
01:53 +46: test/x_test.dart: Feature X should do Y in situation Z2
...
```